compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -209,21 +227,51 @@ class ReplDriver(settings: Array[String],
     compiler
       .compile(parsed)
       .fold(
-        displayErrors,
+        {
+          case (errors: Errors) =>
+            if (!silent) displayErrors(errors)
```

Comment:
I think we want to display error even in silent mode

Reply:
Done